app/src/main/java/com/example/weathertimeapp/mvp/entities/Day.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.weathertimeapp.mvp.entities
+
+import com.example.weathertimeapp.utils.DEFAULT_STRING
+
+data class Day(
+    val main: Main = Main(),
```

Comment:
this is not a representative name to the temperature...please remember that its really important be clear to  correct it.

Reply:
changed, the entity main had the temperatures, now its called Temperature. Thanks